dask/dataframe/tests/test_dataframe.py

```diff
@@ -1145,6 +1145,14 @@ def test_isin():
             d.isin(obj)
 
 
+def test_contains_frame():
+    df = dd.from_pandas(pd.DataFrame({"A": [1, 2], 0: [3, 4]}), 1)
+    assert "A" in df
+    assert 0 in df
+    assert "B" not in df
+    assert 1 not in df
```

Comment:
These already pass for me on master

Reply:
Good catch! That's because python will translate `foo in bar` into something like

```python
for key in bar:
    if key == bar:
        return True

return False
```

as long as `bar` is iterable.

So if we have a wide dataframe, it'll be a bit faster to implement `__contains__`, since it doesn't have to try and fail to find all the keys.

```python
In [5]: %timeit 0 in ddf
1.89 µs ± 27.6 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [6]: %timeit 'a' in ddf
10.8 µs ± 146 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

Not a huge deal in most cases, but we might as well implement it.

Reply:
Thanks for the explanation!